src/main/scala/com.snowplowanalytics/schemaguru/generators/JsonSchemaGenerator.scala

```diff
@@ -242,7 +291,12 @@ object JsonSchemaGenerator {
      * @return JsonSchemaType with recognized properties
      */
     def enrichString(value: String) = {
-      guessFormat(value, formatSuggestions) match {
```

Comment:
Hi @cevaris Thanks for your interest!
Can you provide me some information about usage of "format" for types other than string and with values like "short" or "boolean". AFAIK we can use just date-time, email, hostname, ipv4, ipv6, uri. [Specification](http://json-schema.org/latest/json-schema-validation.html#anchor104) is uncertain about other types, but I can't see any reason why we should add "format: boolean" to "type: boolean". Some [sources](http://spacetelescope.github.io/understanding-json-schema/reference/string.html) claim "format" is only for strings.